rest/route/patch_test.go

```diff
@@ -1052,6 +1078,39 @@ buildvariants:
 	tasks, err = task.Find(task.ByVersion(*respVersion.Id))
 	assert.NoError(t, err)
 	assert.Len(t, tasks, 4)
+
+	// Scheduling a single host task group task should schedule all tasks before it in
+	// the task group
+	patch3 := patch.Patch{
```

Comment:
I left this test even though the logic lives in model, this is to prevent a secret regression somewhere along the line.